docker/Dockerfile.cpu

```diff
@@ -0,0 +1,59 @@
+##############################################################
+# This Dockerfile contains the Devito codes and can be buld using different base images.
+##############################################################
+
+ARG arch=gcc
+
+# Base image 
+FROM python:3.7 as base
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install for basic base not containing it
+RUN apt-get update && apt-get install -y libgl1-mesa-glx
```

Comment:
isn't `libgl1-mesa-glx` only needed by amd? if not, why do we need it?

Reply:
No idea don't remeber where it comes from, removing